test/functional/ex_cmds/grep_spec.lua

```diff
@@ -18,4 +18,20 @@ describe(':grep', function()
     feed('<cr>')  -- Press ENTER
     ok(eval('len(getqflist())') > 9000)  -- IT'S OVER 9000!!1
   end)
+
+  it('works when both shellpipe and makeef are unset', function()
+    if eval("executable('grep')") == 0 then
+      pending('missing "grep" command')
+      return
+    end
+
+    feed_command([[set grepprg=grep\ -r]])
```

Comment:
if command() works that is preferred instead. Not sure why feed_command was used in the existing test.